src/main/java/de/tum/in/www1/artemis/domain/GradingInstruction.java

```diff
@@ -0,0 +1,155 @@
+package de.tum.in.www1.artemis.domain;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+import javax.persistence.*;
+
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
+/**
+ * A Structured Grading Instruction.
+ */
+@Entity
+@Table(name = "grading_instruction")
+@Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
+public class GradingInstruction implements Serializable {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne
+    private Exercise exercise;
+
+    @Column(name = "credits")
+    private double credits;
+
+    @Column(name = "grading_scale")
+    private String gradingScale;
+
+    @Column(name = "instruction_description")
+    @Lob
+    private String instructionDescription;
+
+    @Column(name = "feedback")
+    @Lob
+    private String feedback;
+
+    @Column(name = "usage_count")
+    private int usageCount;
```

Comment:
From your explanation, isn't this more an `applicationLimit` than a count?

Reply:
So "usage_count" is the number instructors enter to indicate when the mistake should be counted as a subsequential mistake.
for example: 0 means every time this mistake is made we should **use** the grading instruction and subtract points.
x means that we should subtract points x times if the mistake occurred more than x times aka we should **use** the grading instruction maximum x times only, that is why I named it usage_count.

Reply:
Ok, so in this case `Integer` would make more sense in this case imo since `null` would be the explicit value if you want to just apply the instruction every time. 
I think you should also add what you just wrote in your comment in the JavaDoc since this is a behavior one can't deduce from just looking at the variable.